server/WebAPI/Controllers/AuthController.cs

```diff
@@ -32,5 +32,21 @@ public IActionResult Auth(AuthRequest authRequest)
             var token = _authService.MakeToken(user);
             return Ok(new AuthResponse { AccessToken = token });
         }
+
+        [AllowAnonymous]
+        [HttpPost]
+        [Route("SignUp")]
+        public IActionResult Registration(AuthRequest authRequest)
```

Comment:
Do you have validation on server?

Reply:
Can I use validation like that? 

`       [Required]
        [EmailAddress]
        [StringLength(50, MinimumLength = 10)]
        public string Email { get; set; }

        [Required]
        [StringLength(50, MinimumLength = 8)]
        public string Password { get; set; }
`

Reply:
Can I use validation in model? Like that

[Required]
[EmailAddress]
[StringLength(50, MinimumLength = 10)]
public string Email { get; set; }

[Required]
[StringLength(50, MinimumLength = 8)]
public string Password { get; set; }

Reply:
yes